internal/step/dummy/provider_test.go

```diff
@@ -29,13 +29,13 @@ func (s *stageChangeHandler) OnStepComplete(
 		panic(fmt.Errorf("invalid previous stage: %s", previousStage))
 	}
 	if previousStageOutputID == nil {
-		panic(fmt.Errorf("no previous stage output ID"))
+		panic(fmt.Errorf("no previous stage output Name"))
```

Comment:
Is this change intentional/correct, or did it get caught in a global replace? Why would a "stage output" message change from "ID" to "Name" because of the deployer terminology change???